### PR TITLE
Feature model

### PR DESCRIPTION
**Pull request by JekxDevil**
Added model for database, using Mongoose

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,72 +1,53 @@
...
-
-var server = require('http').createServer(app);
-
-server.on('listening', function() {
+let server = require('http').createServer(app);
+server.on('listening', function () {
   console.log('Express server listening on port ' + server.address().port);
+
+  const { User } = require('./models/user');
+  new User({ name: 'test', password: 'test', email: '' }).save();
```
Comment (micheledallerive) [inline app.js:47]: Remove pippa

**Review by micheledallerive — CHANGES_REQUESTED** [review]
Fix please. Good job!

**Review by micheledallerive — APPROVED** [review]

**Comment by Hwid2121** [comment]
jef is stupid

**Review by Hwid2121 — DISMISSED** [review]
non vorrei ma devo

**Review by micheledallerive — APPROVED** [review]

**Review by Hwid2121 — APPROVED** [review]
@micheledallerive  stai facendo casini

**Pull request merged**